cardano-api/src/Cardano/Api/TxSubmit/ErrorRender.hs

```diff
@@ -8,31 +8,31 @@ module Cardano.Api.TxSubmit.ErrorRender
 -- They will be defined here for now and then moved where they are supposed to be once they
 -- are working.
 
-import           Cardano.Chain.UTxO.Validation (TxValidationError (..), UTxOValidationError (..))
-import           Cardano.Chain.UTxO.UTxO (UTxOError(..))
 import           Cardano.Chain.Byron.API (ApplyMempoolPayloadErr (..))
+import           Cardano.Chain.UTxO.UTxO (UTxOError (..))
+import           Cardano.Chain.UTxO.Validation (TxValidationError (..), UTxOValidationError (..))
 
 import           Cardano.Prelude hiding ((%))
 
 import           Data.Text (Text)
 import qualified Data.Text as Text
 
-import           Formatting ((%), build, sformat, stext)
+import           Formatting (build, sformat, stext, (%))
 
 renderApplyMempoolPayloadErr :: ApplyMempoolPayloadErr -> Text
 renderApplyMempoolPayloadErr err =
     case err of
-      MempoolTxErr ve -> renderValidationError ve
-      MempoolDlgErr {} -> "Delegation error"
+      MempoolTxErr ve             -> renderValidationError ve
+      MempoolDlgErr {}            -> "Delegation error"
```

Comment:
Vertial alignment (other than that needed by indentation rules) is bad!

Reply:
:sob:

Reply:
I've disabled this rule.